src/fd.zig

```diff
@@ -185,9 +185,9 @@ pub const FDImpl = packed struct {
         if (env.os != .windows or this.kind == .uv) {
             // This branch executes always on linux (uv() is no-op),
             // or on Windows when given a UV file descriptor.
-            const fd = bun.toFD(this.uv());
-            if (fd == bun.STDOUT_FD or fd == bun.STDERR_FD) {
-                log("close({}) SKIPPED", .{this});
+            const fd = this.uv();
+            if (fd == 1 or fd == 2) {
```

Comment:
`bun.STDOUT_FD` was removed here as it may point to the Windows handle. This is more accurate.